src/VisualStudio/VisualStudioInteractiveComponents/Roslyn.VisualStudio.InteractiveComponents.csproj

```diff
@@ -73,11 +65,8 @@
     <NuGetPackageToIncludeInVsix Include="System.Text.Encoding.CodePages" />
     <NuGetPackageToIncludeInVsix Include="System.Runtime.CompilerServices.Unsafe" />
   </ItemGroup>
-  <ItemGroup>
-    <None Include="source.extension.vsixmanifest">
-      <SubType>Designer</SubType>
-    </None>
-  </ItemGroup>
+
+  <Import Project="$(VSToolsPath)\vssdk\Microsoft.VsSDK.targets"/>
 
   <Target Name="VsixSourceItemsOutputGroup"
```

Comment:
Can you delete this bit too?

Reply:
No, this is important. It's called from the referencing project.